cmd/config.go

```diff
@@ -108,7 +108,7 @@ type SMTPConfig struct {
 // it should offer.
 type PAConfig struct {
 	DBConfig   `validate:"-"`
-	Challenges map[core.AcmeChallenge]bool `validate:"gt=0,dive,keys,oneof=http-01 dns-01 tls-alpn-01,endkeys,required"`
+	Challenges map[core.AcmeChallenge]bool `validate:"min=1,dive,keys,oneof=http-01 dns-01 tls-alpn-01,endkeys"`
```

Comment:
Let's actually get rid of `min=1`. It's possible to start boulder with no challenges enabled. You won't be able to create new orders, but you will for instance be able to revoke certificates.